readme.md

```diff
@@ -153,25 +153,7 @@ module.exports = function (req, res) {
 }
 ```
 
-### 2. The `@polka/send-type` library
```

Comment:
This is necessary & was probably the biggest concern/question I got when this was released. Too many thought they had to reach for [`send`](https://www.npmjs.com/package/send) or something. 

The whole point of this block is to show that there'a a built-in dependency you can use for free.

Curious devs would have figured this out, yes, but that's not most.

---

Perhaps the only fix is really to add a number list before these sections, and then link to the sections for more info?

```
There are multiple ways to format or return the server response.<br>
You may mix and match them, as you are not restricted to a particular format in your application(s):

1) foo (link)
2) bar (link)
3) baz (link)
4) bat (link)
```